diff --git a/node_modules/@react-native-async-storage/async-storage/android/build.gradle b/node_modules/@react-native-async-storage/async-storage/android/build.gradle
index 4560dc7..fd3033f 100644
--- a/node_modules/@react-native-async-storage/async-storage/android/build.gradle
+++ b/node_modules/@react-native-async-storage/async-storage/android/build.gradle
@@ -37,32 +37,20 @@ if (isNewArchitectureEnabled) {
 }
 
 android {
-    def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
-    if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
-        namespace "com.reactnativecommunity.asyncstorage"
-        buildFeatures {
-            buildConfig true
-        }
-    }
-
-    compileSdkVersion project.ext.AsyncStorageConfig.compileSdkVersion
-    // Used to override the NDK path/version by allowing users to customize
-    // the NDK path/version from their root project (e.g. for M1 support)
-    if (rootProject.hasProperty("ndkPath")) {
-        ndkPath rootProject.ext.ndkPath
-    }
-    if (rootProject.hasProperty("ndkVersion")) {
-        ndkVersion rootProject.ext.ndkVersion
+    namespace "com.reactnativecommunity.asyncstorage"
+    buildFeatures {
+        buildConfig true
     }
 
+    compileSdkVersion 36
 
     defaultConfig {
-        minSdkVersion project.ext.AsyncStorageConfig.minSdkVersion
-        targetSdkVersion project.ext.AsyncStorageConfig.targetSdkVersion
-        buildConfigField "Long", "AsyncStorage_db_size", "${project.ext.AsyncStorageConfig.databaseSizeMB}L"
-        buildConfigField "boolean", "AsyncStorage_useDedicatedExecutor", "${project.ext.AsyncStorageConfig.useDedicatedExecutor}"
-        buildConfigField "boolean", "AsyncStorage_useNextStorage", "${useNextStorage}"
-        buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", "${isNewArchitectureEnabled}"
+        minSdkVersion 24
+        targetSdkVersion 36
+        buildConfigField "Long", "AsyncStorage_db_size", "6L"
+        buildConfigField "boolean", "AsyncStorage_useDedicatedExecutor", "false"
+        buildConfigField "boolean", "AsyncStorage_useNextStorage", "false"
+        buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", "false"
     }
     lintOptions {
         abortOnError false
